src/main/webapp/app/exam/participate/summary/exam-participation-summary.component.ts

```diff
@@ -28,11 +31,13 @@ export class ExamParticipationSummaryComponent implements OnInit {
     readonly AssessmentType = AssessmentType;
     readonly IncludedInOverallScore = IncludedInOverallScore;
     readonly SUBMISSION_TYPE_ILLEGAL = SubmissionType.ILLEGAL;
+    readonly PlagiarismVerdict = PlagiarismVerdict;
 
     /**
      * Current student's exam.
      */
     private _studentExam: StudentExam;
+    plagiarismCaseInfos: { [exerciseId: number]: PlagiarismCaseInfo } = {};
```

Comment:
😍Cool: I love the use of indexable type here. Will have to remember that! Gj

Reply:
Thank you very much! 😍